book/guides/team_docs.md

```diff
@@ -36,3 +36,4 @@ Unfortunately we do not have a technical way to implement per user quotas, so we
 
 - How much space is left on the [entire shared volume](https://grafana.leap.2i2c.cloud/d/hub-dashboard/jupyterhub-dashboard?orgId=1&viewPanel=23)? If the free space falls *below 20%* we need to discuss further actions.
 - [Which users are taking up more than the allowed 50GB](https://grafana.leap.2i2c.cloud/d/bd232539-52d0-4435-8a62-fe637dc822be/home-directory-usage-dashboard?orgId=1). For users who take up more than 50GB the suggested action is to reach out via slack or email.
+- To send emails, a token file is setup as the OAUTH_GMAIL_CREDENTIALS Github Secret in the member management repo; every so often the action will require re-authentication, generating a new token that should replace the existing secret. This makes use of OAUTH_GMAIL_CLIENT_SECRET, which never needs to change
```

Comment:
Nice. Can you add some more detailed instructions how to regenerate the creds? This is not entirely clear to me. Also we should point to the specific repo and action in the future and link them here.